src/strings.ts

```diff
@@ -137,6 +137,15 @@ export const FiscalCode = PatternString(
 
 export type FiscalCode = t.TypeOf<typeof FiscalCode>;
 
+/**
+ * A fake Fiscal Number (for persons)
+ */
+export const FiscalCodeFake = PatternString(
```

Comment:
I'd like to use a consistent naming for things that are "fake" and can be used for testing and development

```suggestion
export const SandboxFiscalCode = PatternString(
```